flipperdevices/flipperzero-firmware
# [FL-781] FURI, CLI, stdlib: stdout hooks, integration between subsystems, uniform printf usage

# What's new

- Furi stdglue: now hookable and thread/ISR aware
- Cli: use stdout hooks and migrate to printf 
- Applications: uniform line endings
- Power: prevent sleep till core 2 is ready
- Cli commands: log - access global logging
- Fix idle task stack damage caused by systicks (FL-781)
- BLE insomnia while active connection 

# Verification

- Compile and upload
- Connect to flipper cli
- Use `log` command
- Run any other application that uses printf, verify that output is shown in cli

# Checklist (do not modify)
